qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -138,11 +150,19 @@ def test_all(self):
         self.assertIn('bucket_info', data)
         data = data['bucket_info']
         self.assertIn('bucket', data)
+        self.assertTrue(isinstance(data['bucket'], dict))
         self.assertIn('quota', data)
+        self.assertTrue(isinstance(data['quota'], dict))
         self.assertIn('creation_time', data)
+        self.assertTrue(isinstance(data['creation_time'], basestring))
```

Comment:
And to avoid asserting that all items exist, maybe taking a more pythonic approach (assuming they all exist but handling when they don't):

```python
def test_all(self):
    try:
     self.assertIsInstance(data['bucket'], dict)
   except KeyError as e:
     self.fail(...)
```

Reply:
```suggestion
        self.assertSchema(data['bucket'],str)
```
should work ootb. And supports all [string types](https://github.com/ceph/ceph/blob/master/qa/tasks/mgr/dashboard/helper.py#L396)

Reply:
Great to know that it exists this @sebastian-philipp! In that case, wouldn't it be better to validate the whole data structure at once?:

```python
bucket_schema = {
    'creation_time': str,
    #<other_fields>: <data_types>,
  ....
  }

self.assertSchema(data, bucket_schema)
```